feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+# PF-1.3: Policy-based Static GUE Encapsulation to IPv4 tunnel
+
+## Summary
+
+This test verifies Policy Forwarding (PF) action to encapsulate packets in an
+IPv4 GUE tunnel. The encapsulation is based on a statically configured GUE
+tunnel and static route.
+
```

Comment:
It would be worth linking to the OpenConfig configuration that specifies this feature, or calling out some of the behavioural properties that are only implicitly stated at the moment.

Enumerating a couple:

* when the DSCP for the tunnel is not explicitly set, the assumption seems to be that the DSCP is copied from the inner to the outer header (i.e., PF-1.3.1 assumes that TOS `0x80` on the inner results in TOS `0x80` on the outer).
* similar to the above, the expectation for TTL seems to be that when not explicitly specified, TTL/hop limit is copied from inner to the outer encapsulating header.

Reply:
The OpenConfig configuration is a TODO in the yaml section below and will be sent as a proposal in a follow up pull request based on other comment from Darren. However, I've added these notes to the beginning of the README.